### PR TITLE
Added article "High contrast simulation"

### PR DESCRIPTION
**Pull request by codepo8**
**Rendered article for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/accessibility/forced-color-simulation?branch=pr-en-us-806

This PR adds documentation on high contrast simulation, due to land next week in Dev Channel. - codepo8, July 9, 2020.

November 2, 2021: merged from master into this PR/branch, resolved a minor "merge conflict" that wasn't really a conflict, just adjacent lines changed in the TOC file.

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/accessibility/forced-color-simulation.md [hunk 1]
```diff
@@ -1,22 +1,23 @@
 ---
-title: Display the current web site in high contrast mode (CSS Forced Color)
+description: Display the current web site in high contrast mode (CSS Forced Color)
+title: High Contrast simulation
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 06/18/2020
+ms.date: 07/16/2020
```
[the remark this answers is not in the document]
Comment (codepo8) [reply microsoft-edge/devtools-guide-chromium/accessibility/forced-color-simulation.md]: WFM

File: microsoft-edge/devtools-guide-chromium/accessibility/forced-color-simulation.md [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
 ms.topic: article
 ms.prod: microsoft-edge
 keywords: microsoft edge, web development, f12 tools, devtools
 ---
 
 # High Contrast simulation  
 
-Windows and other Operating Systems have a [setting that allows users to show the interface with a set of reduced colors][WindowsHighContrastMode]. These could be either pre-set color schemes or defined by the user. This helps people with several needs: those with limited vision, those suffering from Migraine, [Irlen Syndrome][IrlenSyndromeLink] or [Dyslexia][DyslexiaLink]. 
+Windows and other operating systems \(OSs\) contain a [setting that allows users to show the interface with a set of reduced colors][WindowsHighContrastMode].  The setting may be either pre-set color schemes or defined by the user.  The setting helps people with several needs including those with limited vision, suffering from migraines, suffering from [Irlen Syndrome][WikiIrlenSyndrome], or living with [Dyslexia][WikiDyslexia].
```
[the remark this answers is not in the document]
Comment (codepo8) [reply microsoft-edge/devtools-guide-chromium/accessibility/forced-color-simulation.md:14]: LGTM

**Comment by jm-trd-ms** [comment]
Caught up content from primary branch.

File: microsoft-edge/devtools-guide-chromium/accessibility/forced-color-simulation.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+             Turn off high contrast simulation using the command menu  
+          :::image-end:::  
+       :::column-end:::
+    :::row-end:::  
+
+View and change your CSS like any other web page.  For more information, see [Get Started With Viewing And Changing CSS][DevtoolsGuideChromiumCssIndex].  
+
+<!-- links --> 
+
+[DevtoolsGuideChromiumMain]: ../../devtools-guide-chromium.md "Microsoft Edge (Chromium) Developer Tools | Microsoft Docs"
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/accessibility/forced-color-simulation.md:87]:
```suggestion
[DevtoolsGuideChromiumMain]: ../index.md "Microsoft Edge (Chromium) Developer Tools | Microsoft Docs"  
```
According to my calculations, go up 1 dir, then open index file.  Check this link after commit & build.  To fix build Warning.

File: microsoft-edge/devtools-guide-chromium/accessibility/forced-color-simulation.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+Windows and other operating systems \(OSs\) contain a [setting that allows users to show the interface with a set of reduced colors][WindowsHighContrastMode].  The setting is called **High Contrast Mode**.  The setting may use either pre-set color schemes or settings that are defined by the user.  The setting helps people with several needs including those with limited vision, suffering from migraines, suffering from [Irlen Syndrome][WikiIrlenSyndrome], or living with [Dyslexia][WikiDyslexia].  
+
+The exact numbers are not available, but roughly 4% of all Windows users use display settings with reduced color settings.  According to the [2018 Survey of Users with low Vision][WebaimProjectsLowvisionsurvey2Contrastmode] provided by WebAIM, more than half of low vision users use **High Contrast Mode**.  
+
+To make it easier for you \(the developer\) to test how your products appear in **High Contrast Mode**, [Microsoft Edge DevTools][DevtoolsGuideChromiumMain] include a way to simulate it without changing your Operating System settings.  The simulation also helps users of other OSs that do not have the same functionality and therefore are not able to test it.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/accessibility/forced-color-simulation.md:18]:
```suggestion
To make it easier for you \(the developer\) to test how your products appear in **High Contrast Mode**, [Microsoft Edge DevTools][DevtoolsGuideChromiumMain] includes a way to simulate it without changing your Operating System settings.  The simulation also helps users of other OSs that do not have the same functionality and therefore are not able to test it.  
```

**Comment by mikehoffms** [comment]
@codepo8 @Reezaali 
This PR adds the article "High contrast simulation", but that feature isn't present in the Rendering tool.
Temporarily Closing this PR (1 year 4 months old) without merging/publishing it.  Hoping to later Reopen this PR and eventually merge/publish it.  Keeping the "on-hold" label.  This PR is "For future reference"; someone may reference this PR in the future.

**Pull request closed without merging**